src/cdk/schematics/update-tool/utils/parse-tsconfig.ts

```diff
@@ -12,6 +12,9 @@ import {FileSystemHost} from './virtual-host';
 import {dirname} from 'path';
 import {formatDiagnostics} from './diagnostics';
 
+/** Code of the error raised by TypeScript when a tsconfig doesn't match any files. */
+const NO_INPUTS_ERROR_CODE = 18003;
```

Comment:
This is not exposed right?

Reply:
Nope, it looks like TS doesn't expose an enum of their error codes.